perma_web/perma/templates/docs/developer/index.html

```diff
@@ -37,7 +37,7 @@ <h3 id="api-key" class="body-bh">API Key</h3>
         (or navigating to the Tools section, which can be found in your Dashboard under the Settings section).
         Click on the <i>Generate an API key</i> button.</p>
 
-      <img src="{{ STATIC_URL }}img/docs/create-api-key.png" class="img-responsive" alt="Create an API key "/>
+      <img src="{{ STATIC_URL }}img/docs/create-api-key.png" class="img-responsive" alt="Screenshot of Settings > Tools page. A button labelled 'Generate an API key' appears near the bottom of the page."/>
```

Comment:
Change "labelled" to "labeled" to match U.S. usage per http://grammarist.com/spelling/label/ and our privacy policy: https://github.com/harvard-lil/perma/blob/4598a738bc9a0536109a81e5184173635dcfefd5/perma_web/perma/templates/privacy_policy.html#L68

Reply:
Thanks. I always get mixed up because of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute